migrations/20200915113240_van-sync-task.js

```diff
@@ -111,7 +111,8 @@ exports.up = function(knex) {
               'job_request_id', v_job_request_id,
               'contact_count', v_contact_count
             )
-          )
+          ),
+          'van-api'
```

Comment:
This could cause a refresh to be delayed while a 100k sync runs - let's have:
`van-api-refresh` and `van-api-sync`